setup.py

```diff
@@ -1,3 +1,4 @@
+from git import Repo
```

Comment:
Kill this since you are importing git in the git_version function with a try: except: if the module is missing